src/components/ErrorHandling/Sentry.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  ErrorBoundary as SentryErrorBoundary,
+  init,
+  captureException,
+  withScope,
+} from "@sentry/react";
+import type { Scope, SeverityLevel } from "@sentry/react";
+import { environmentVariables } from "utils";
+import ErrorFallback from "./ErrorFallback";
+
+const { getReleaseStage, getSentryDSN, isProduction } = environmentVariables;
+
+const initializeSentry = () => {
+  const releaseStage = getReleaseStage() || "development";
+  try {
+    init({
+      dsn: getSentryDSN(),
+      debug: !isProduction(),
+      normalizeDepth: 5,
+      release: APP_VERSION,
+      environment: releaseStage,
+    });
+  } catch (e) {
+    console.error("Failed to initialize Sentry", e);
+  }
+};
+
+const isInitialized = () => {
+  // @ts-expect-error
+  const globalSentry = window.__SENTRY__; // eslint-disable-line no-underscore-dangle
+
+  // If there is a global __SENTRY__ that means that in any of the callbacks init() was already invoked
+  // https://github.com/getsentry/sentry-javascript/blob/e5e6a6bd8ab2bbec59879971595a7248fa132826/packages/browser/src/loader.js#L116-L121
+  return (
+    !(typeof globalSentry === "undefined") &&
```

Comment:
Would this be the same?
```suggestion
    typeof globalSentry !== "undefined" &&
```